### ISSUE TITLE
Implement Spanner.attach() and Spanner.detach()

### ISSUE THREAD

**Issue by trevorbaca** [Abjad/abjad#78]
_From [trevorb...@gmail.com](https://code.google.com/u/118334091018461915321/) on March 20, 2012 11:01:59_

It would be nice to unify spanner attachment syntax and mark attachment syntax to whatever extent is sensible.

It seems like a good way to bookmark the conversation is to propose Spanner.attach() and Spanner.detach() and then discuss from there when time allows.

_Original issue: http://code.google.com/p/abjad/issues/detail?id=77_
